packages/node_modules/@webex/plugin-meetings/src/roap/util.js

```diff
@@ -51,11 +51,12 @@ RoapUtil.compareLastRemoteOffer = (currentOffer, previousOffer) => {
     previousOffer.forEach((sdp, index) => {
       index1 = sdp.indexOf(_IN_);
       index2 = currentOffer[index].indexOf(_IN_);
-
       if (sdp.substring(index1) !== currentOffer[index].substring(index2)) {
         difference = true;
       }
     });
+
+    // return difference;
```

Comment:
?

Reply:
There is still a bigger bug we need to fix related to the ROAP . We are seeing an issue with the Chrome offer after answer

Reply:
Can you add a TODO?

Reply:
done